### PR TITLE
Add infrastructure nodes sizing recommendations for 4.6

### PR DESCRIPTION
**Pull request by mohit-sheth**
This commit adds information on infrastructure nodes which hosts
Router, Registry, Logging and Monitoring components for various
nodes scale. The data is based on the cluster maximums and density
focused scale test runs.

### CODE REVIEW

**Comment by mohit-sheth** [comment]
@ahardin-rh @jtaleric @mffiedler @chaitanyaenr PTAL.

File: modules/infrastructure-node-sizing.adoc [hunk 1]
```diff
@@ -33,7 +62,7 @@ The infrastructure node resource requirements depend on the cluster age, nodes,
 ====
 These sizing recommendations are based on scale tests, which create a large number of objects across the cluster. These tests include reaching some of the cluster maximums. In the case of 250 and 500 node counts on a {product-title} {product-version} cluster, these maximums are 10000 namespaces with 61000 pods, 10000 deployments, 181000 secrets, 400 config maps, and so on. Prometheus is a highly memory intensive application; the resource usage depends on various factors including the number of nodes, objects, the Prometheus metrics scraping interval, metrics or time series, and the age of the cluster. The disk size also depends on the retention period. You must take these factors into consideration and size them accordingly.
 
-The sizing recommendations are applicable only for the infrastructure components which gets installed during the cluster install - Prometheus, Router and Registry. Logging is a day two operation and the recommendations do not take it into account.
+The sizing recommendations are applicable for the infrastructure componnets - Prometheus, Router, Registry and Logging.
```
Comment (chaitanyaenr) [inline modules/infrastructure-node-sizing.adoc:65]: s/componnets/components

**Review by chaitanyaenr — APPROVED** [review]
LGTM

File: modules/infrastructure-node-sizing.adoc [hunk 2]
```diff
@@ -7,21 +7,53 @@
 
 The infrastructure node resource requirements depend on the cluster age, nodes, and objects in the cluster, as these factors can lead to an increase in the number of metrics or time series in Prometheus. The following infrastructure node size recommendations are based on the results of cluster maximums and control plane density focused testing.
 
+[IMPORTANT]
+====
+The sizing recommendations below are applicable only for the infrastructure components which gets installed during the cluster install - Prometheus, Router and Registry. Logging is a day two operation and the recommendations do not take it into account.
```
Comment (ahardin-rh) [inline modules/infrastructure-node-sizing.adoc:12]: The sizing recommendations below are only applicable for the Prometheus, Router, and Registry infrastructure components, which are installed during cluster installation. Logging is a day-two operation and is not part of these recommendations.

File: modules/infrastructure-node-sizing.adoc [hunk 3]
```diff
@@ -7,21 +7,53 @@
...
+| 500
 | 32
-| 192
+| 128
+
+|===
+
+[IMPORTANT]
+====
+The sizing recommendations below are applicable for the infrastructure components - Prometheus, Router, Registry and Logging.
```
Comment (ahardin-rh) [inline modules/infrastructure-node-sizing.adoc:39]: for the Prometheus, Router, Registry, and Logging infrastructure components.

**Review by mffiedler — COMMENTED** [review]
@ahardin-rh Can we get a preview build?  The rich diff is  a little confusing.  @mohit-sheth It looks like we are adding a second table.  What's the difference between them?   The addition of logging?

**Comment by ahardin-rh** [comment]
@mffiedler Yeah, it looks like the preview build did not automatically pop up as expected. I can fetch Mohit's branch and build a preview manually. Please hold :smile:

**Comment by ahardin-rh** [comment]
@vikram-redhat FYI the preview build did not work for this PR; not sure if there's something wrong with the tooling.

**Comment by mohit-sheth** [comment]
>@mohit-sheth It looks like we are adding a second table. What's the difference between them? The addition of logging?

@mffiedler Yes

**Comment by ahardin-rh** [comment]
@mffiedler Preview Build: http://file.rdu.redhat.com/~ahardin/01-08-2021/scalability_and_performance/recommended-host-practices.html#infrastructure-node-sizing_

**Comment by vikram-redhat** [comment]
The preview build only works for acked submitters. @mohit-sheth is not in that list. If he will be submitting more PRs, I can add him.

**Comment by chaitanyaenr** [comment]
@vikram-redhat, @mohit-sheth will be submitting more PR's in the future with help from @ahardin-rh, can we add him to the acked submitters please?

**Comment by vikram-redhat** [comment]
> @vikram-redhat, @mohit-sheth will be submitting more PR's in the future with help from @ahardin-rh, can we add him to the acked submitters please?

Done. New PRs from @ @mohit-sheth will now have the auto preview.

**Comment by ahardin-rh** [comment]
Thanks, @vikram-redhat!

File: modules/infrastructure-node-sizing.adoc [hunk 4]
```diff
@@ -7,21 +7,53 @@
...
 
 | 250
+| 16
+| 128
+
+| 500
 | 32
-| 192
+| 128
+
```
Comment (mffiedler) [inline modules/infrastructure-node-sizing.adoc:34]: Sorry to be nit-picky but I don't like the two tables back-to-back with no description of the second table.   @ahardin-rh feel free to overrule me on this.   In my opinion, a better approach might be a single table where the core and memory columns have two values:  one without logging and one with logging.    Opinions everyone?

Comment (ahardin-rh) [reply modules/infrastructure-node-sizing.adoc:34]: That seems like a good suggestion to me!

**Comment by ahardin-rh** [comment]
@mohit-sheth Any update on this? Thanks!

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**